### DISCUSSION TITLE
Reconfiguring keyboard controls

### DISCUSSION THREAD

**Discussion by bhattsiddharth** [open-goal/jak-project#2759] [Q&A] [answered]
So, I don't have a functioning PS2 controller anymore and I want to play Jak and Daxter using keyboard and mouse. I can't figure out how to reconfigure the keybinds to more comfortable settings. There is a thread on r/jakanddaxter but the sub seems to be indefinitely locked. Someone please help

**Comment by Zedb0T** [comment]
https://github.com/open-goal/jak-project/issues/2737#issuecomment-1595309168

**Comment by xTVaser** [comment, marked as answer]
This month's release will add a new way to easily re-assign controller, keyboard, and mouse keybindings from the game's pause menu.

**Comment by Zedb0T** [comment]
https://github.com/open-goal/jak-project/assets/89345505/66b74686-ba3e-475d-8f70-e94a72f3d108
